client-report/gulpfile.js

```diff
@@ -324,12 +325,12 @@ function deploy(uploader) {
 
 function doUpload() {
   var uploader;
-  if ('s3' === polisConfig.UPLOADER) {
+  if ('s3' === config.get('uploader')) {
```

Comment:
FWIW It doesn't have to happen as part of this pull request, but I think we're going to removing the `doUpload` pathway from the individual client gulpfiles. Gulp (or whatever it's replaced with) should _just_ build the clients. Deploying compiled assets should be a separate step, either as part of the docker-compose setup, or as a standalone step in some other process (e.g. if serving via s3-backed CDN, as we do; again see #1301 for recent context on this).